templates/default/quicksearch-indicators-panel.html

```diff
@@ -11,6 +11,13 @@
 		</div>
 	</div>
 	{/block}
+    <div style="margin-left: 10px;">
+            <script type="text/javascript">
+            if (!window.screenTop && !window.screenY) {
+                document.write("{$instance_label}");
+            }
+            </script>
```

Comment:
Co to miało robić? :)

Reply:
To jest pokazywane tylko gdy jest tryb "na pełnym ekranie"